src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -199,6 +222,81 @@ private static string GetMemberElementName(FunctionDescriptor member)
             return String.Format("{0}:{1}", prefixCode, memberName);
         }
 
+        private enum XmlTagType
+        {
+            None,
+            Member,
+            Summary,
+            Parameter,
+            SearchTags
+        }
+
+        private static void LoadDataFromXml(XmlReader reader)
+        {
+            if (reader == null)
+                return;
+
+            MemberDocumentNode currentDocNode = new MemberDocumentNode();
```

Comment:
`member` attribute is always the first one. But if I leave it without initialization, i'll get error, that I use unassigned value.